core/src/main/java/org/infinispan/interceptors/impl/EntryWrappingInterceptor.java

```diff
@@ -583,9 +602,9 @@ public Object visitPutMapCommand(InvocationContext ctx, PutMapCommand command) t
          if (newMap.size() > 0) {
             PutMapCommand clonedCommand = commandFactory.buildPutMapCommand(newMap,
                   command.getMetadata(), command.getFlagsBitSet());
-            invokeNextInterceptor(ctx, clonedCommand);
+            ctx.forkInvocationSync(clonedCommand);
          }
-         return null;
+         return ctx.continueInvocation();
```

Comment:
Actually, the return value isn't used, so I should have kept `return null`. In fact, I thought I already reverted this ;)